File: GeeksForGeeks-Problems/JAVASCRIPT/Functional Programming/Binary Number to Decimal Number.js
// User function Template for javascript

class Solution {
  binaryToDecimal(b) {
    // write your code here
    let j = 0;
    let ans = 0;
    for (let i = b.length - 1; i >= 0; i--) {
      ans += parseInt(b[i]) * Math.pow(2, j);
      j++;
    }
    return ans;
  }
}

File: GeeksForGeeks-Problems/CPP/String/Binary Number to Decimal Number.cpp
class Solution
{
public:
    int binaryToDecimal(string &b)
    {
        // Code here.
        int j = 0;
        int ans = 0;
        for (int i = b.length() - 1; i >= 0; i--)
        {
            ans += (b[i] - '0') * pow(2, j);
            j++;
        }
        return ans;
    }
}; 